server/src/main/java/org/jetbrains/bsp/bazel/server/ServerInitializer.java

```diff
@@ -68,20 +68,23 @@ public static void main(String[] args) {
   }
 
   private static BazelBspServerConfig getBazelBspServerConfig(Path bspProjectRoot, String[] args) {
-    var pathToBazel = args[0];
     var projectView = getProjectView(bspProjectRoot, args);
+    var pathToBazel = projectView.getBazelPath().get().getValue();
```

Comment:
`.get()` is ok here? Also line 79.

Reply:
yea, if it is empty, test will fail anyway (I remember that there was some implicit for it in scala)